src/pages/login.js

```diff
@@ -0,0 +1,80 @@
+import GoogleButton from '@/components/auth/GoogleButton';
+import NavTab from '@/components/auth/NavTab';
+import PrimaryButton from '@/components/auth/PrimaryButton';
+import InputGroup from '@/components/auth/inputGroup';
+import AuthLayout from '@/layouts/auth';
+import loginSchema from '@/lib/loginSchema';
+import styles from '@/styles/auth.module.scss';
+import { useFormik } from 'formik';
+import Head from 'next/head';
+import { MdOutlineLock, MdOutlineMailOutline } from 'react-icons/md';
+
+const Login = () => {
+  const formik = useFormik({
+    initialValues: {
+      email: '',
+      password: '',
+    },
+    validationSchema: loginSchema,
+    onSubmit,
+  });
+
+  async function onSubmit(values) {
+    console.log(values);
+  }
+
+  return (
+    <>
+      <Head>
+        <title>Beeus</title>
```

Comment:
we could give a name specific for the pages with titles, example
```suggestion
        <title>Tela de login | Beeus </title>
```